docs/pdk.md

```diff
@@ -0,0 +1,32 @@
+## Puppet Development Kit
+
+The Puppet Development Kit (PDK) is a package of development and testing tools to help you create great Puppet modules.
+
+The PDK includes key Puppet code development and testing tools for Linux, Windows, and Mac OS X workstations, so you can install one package with the tools you need to create and validate new modules. PDK includes testing tools, a complete module skeleton, and command line tools to help you create, validate, and run tests on Puppet modules. PDK also includes all dependencies needed for its use.
+
+PDK includes the following tools:
+
+Tool   | Description
+----------------|-------------------------
+pdk | Command line tool for generating and testing modules
+rspec-puppet | Tests the behavior of Puppet when it compiles your manifests into a catalog of Puppet resources.
+puppet-lint | Checks your Puppet code against the recommendations in the Puppet Language style guide.
+puppet-syntax | Checks for correct syntax in Puppet manifests, templates, and Hiera YAML.
+metadata-json-lint | Validates and lints `metadata.json` files in modules against  Puppet's module metadatastyle guidelines.
+rspec-puppet-facts | Adds support for running rspec-puppet tests against the facts for your supported operating systems.
+puppetlabs_spec_helper | Provides classes, methods, and Rake tasks to help with spec testing Puppet code.
+
+
+## Getting started
+
+To get started, install the PDK, and then create and test a module.
+
+<!--TK: overview workflow graphic-->
+
+1. [Generate a module](./pdk_generating_modules.md) using the `pdk new module` command.
+1. [Validate your module](./pdk_testing.md), to verify that it is well-formed.
+1. [Unit test](./pdk_testing.md) your module, to verify that all dependencies and directories are present.
+1. [Generate a class]](./pdk_generating_modules.md) for your module, using the `pdk new class` command.
```

Comment:
Extra `]` here, also should the link be to `./pdk_generating_modules.md#generate-a-new-class`